README.md

```diff
@@ -9,7 +9,6 @@
 ``` bash
 # install all dependencies
 $ cd styleguide && yarn install && cd ..
-$ yarn run styleguide:build
```

Comment:
@visualjerk Don`t we need to run it once to get the tokens?

Reply:
No, because the tokens are also build inside the styleguides `dev` command:
```
"dev": "npm-run-all --parallel theo:onchange theo servedev",
```

So we only need to run `yarn styleguide:build` when building for production directly inside a fresh installation.

Reply:
we need to tell new devs that they need to run the styleguide at least once or not?

Reply:
Damn it, you're right :D I was assuming everyone will start `yarn styleguide` before developing, but that is obviously not true. So we should keep the build step in there.